platform/ios/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ Mapbox welcomes participation and contributions from everyone. Please read [CONT
 
 * A new runtime styling API allows you to adjust the style and content of the base map dynamically. All the options available in [Mapbox Studio](https://www.mapbox.com/studio/) are now exposed via MGLStyle and subclasses of MGLStyleLayer and MGLSource. ([#5727](https://github.com/mapbox/mapbox-gl-native/pull/5727))
 * The user location annotation is now customizable via the newly added `MGLUserLocationAnnotationView` class. ([#5882](https://github.com/mapbox/mapbox-gl-native/pull/5882))
+* Xcode 7.3 or higher is now officially required to build and integrate this SDK. ([#6059](https://github.com/mapbox/mapbox-gl-native/issues/6059))
```

Comment:
Nit: remove “officially”. Also, “install” or “link against” would be more relevant to most readers than “build and integrate”.